examples/text-generation/utils.py

```diff
@@ -285,11 +289,17 @@ def peft_model(args, model_dtype, logger, **model_kwargs):
 
         model = AutoModelForCausalLM.from_pretrained(args.model_name_or_path, torch_dtype=model_dtype, **model_kwargs)
         model = PeftModel.from_pretrained(model, args.peft_model, torch_dtype=model_dtype, **model_kwargs)
+    if hasattr(model, "merge_and_unload"):
```

Comment:
Do you have an example of model that doesn't have the `merge_and_unload` method?

Reply:
prompt tuning/p tuning and prefix tuning do not have the merged_and_unload method